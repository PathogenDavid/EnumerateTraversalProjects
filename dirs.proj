<Project Sdk="Microsoft.Build.Traversal">
  <ItemGroup>
    <ProjectReference Include="*/dirs.proj" />
  </ItemGroup>
</Project>
