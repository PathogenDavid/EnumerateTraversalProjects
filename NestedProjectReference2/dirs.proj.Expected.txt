NestedProjectReference2/Application/Application.csproj
NestedProjectReference2/ClassLibrary/ClassLibrary.csproj
NestedProjectReference2/ClassLibrary2/ClassLibrary2.csproj
